docs/zarr-v3.md

```diff
@@ -0,0 +1,96 @@
+# zarr-v3 Guide/Roadmap
+
+`anndata` now uses the much improved {mod}`zarr` v3 package and also allows writing of datasets in the [v3 format], with the exception of structured arrays.
+Users should notice a significant performance improvement, especially for cloud data, but also likely for local data as well.
+Here is a quick guide on some of our learnings so far:
+
+## Remote data
+
+We now provide the {func}`anndata.experimental.read_lazy` feature for reading as much of the {class}`~anndata.AnnData` object as lazily as possible, using `dask` and {mod}`xarray`.
+Please note that this feature is experimental and subject to change.
+To enable this functionality in a performant and feature-complete way for remote data sources, we use [consolidated metadata] on the `zarr` store (written by default).
+Please note that this introduces consistency issues - if you update the structure of the underlying `zarr` store i.e., remove a column from `obs`, the consolidated metadata will no longer be valid.
```

Comment:
Please add some guidance here: how *can* one remove a column from `obs` after this? Also which operations update the structure? Can you e.g. add a column? I know that this is about remote data, which can currently not be written over https (right?) but this is still probably good to mention.

Also please use actual dashes (<kbd>Alt/option</kbd>+<kbd>-</kbd>) for dashes, not minus/hyphens.

```suggestion
Please note that this introduces consistency issues – if you update the structure of the underlying `zarr` store i.e., remove a column from `obs`, the consolidated metadata will no longer be valid.
```